hr_leave_validator_responsive_follow/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 (https://www.gnu.org/licenses/agpl-3.0.html)
+{
+    "name": "Hr Leave Validator Responsive Follow",
```

Comment:
Typo (also for addon name):
```suggestion
    "name": "Hr Leave Validator Responsible Follow",
```